### ISSUE TITLE
[BUG] Cannot login my Youtube account

### ISSUE THREAD

**Issue by quigonjinn28** [rivafarabi/deckboard#668]
Describe the bug
When I try to login my Youtube account for the first time, the Youtube login button doesn't do anything.

To Reproduce
1. Go to Settings
2. Click on Accounts -> Youtube Login
3. Nothing happen

Expected behavior
As a user I would expect that the application pop up a login interface to my Youtube account as the other login buttons do.

Server App
 - OS: Windows
 - Version [1.9.86-1.9.87]

Required files
[Log&Database.zip](https://github.com/rivafarabi/deckboard/files/6096037/Log.Database.zip)

Additional context
I first downloaded the application was on the version 1.9.86 so I cannot be sure from where this issue is happening

**Comment by enocvt** [comment]
Same here ✋

**Comment by Ratjack** [comment]
Yup, same here.  All other buttons work but the YouTube login button has the animation like it's being clicked, but nothing happens.
